tests/config/test_data_config.py

```
@@ -0,0 +1,110 @@
+    """Test that DataConfig initializes correctly with default values."""
+    with pytest.raises(
+        TypeError,
+        match="missing .* required positional argument.*train_labels_path.*val_labels_path",
+    ):
+        DataConfig()
+
+
+def test_preprocessing_config_initialization():
+    """Test PreprocessingConfig with valid values."""
+    with pytest.raises(ValueError, match="scale"):
+        config = PreprocessingConfig(max_height=256, max_width=256, scale=(0.5, 0.5))
```

Ruff (0.8.2)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)